index.css

```diff
@@ -1,361 +1,179 @@
+    color: var(--secondary-color);
     font-size: 16px;
-}
-.menu-item2 :hover{
-    color: #eeda5f;
-    
-    
-}
-.menu-item2{
+  }
+  
+  .menu-item2:hover {
+    color: var(--secondary-color);
+  }
+  
+  .menu-item2, 
+  .menu-item1 {
     font-size: 16px;
-    
-}
-.menu-item3 a{
-color: #2cd655;
-font-size: 18px;
-
-}
-/*layout*/
-.grid-container {
-    display: grid;
-    grid-template-columns: auto auto auto ;
-    grid-gap:10px;
-    background-color:white;
-    padding-top: 30px;
-    padding-left: 15px;
   }
-.grid-container{
+  
+  .menu-item3 a {
+    color: var(--primary-color);
+    font-size: 18px;
+  }
+  
+  /* Layout chung */
+  .grid-container {
+    display: grid;
+    grid-template-columns: repeat(3, auto);
+    gap: 10px;
+    background-color: white;
+    padding: 30px 15px 20px;
     width: 1167px;
     margin: auto;
-    padding-bottom: 20px;
-} 
+  }
+  
   .grid-container > div {
     background-color: rgba(255, 255, 255, 0.8);
-    
   }
-.grid-item1{
-    grid-column: 1/3;
-    border: 1px solid orange;
-    
-}
-.grid-item1{
-     width: 847px;
-     height: 312,8px;
-}
-.grid-item1 img{
-    width: 847px;
-    
-}
-.grid-item2{
-    grid-column: 3;
-    grid-row: 1 / span 5;
-    
-}
-.grid-item2{
-    width: 271px;
-    height: 1050px ;
-}
-.grid-item20{
-    border: 1px solid orange;
-    border-radius: 30px 30px 0px 0px;
-}
-.grid-item21{
-    background-color: #25ac45;
-    text-align: center;
-    font-size: 14px;
-    padding: 10px;
-    color: white;
-    border-radius: 20px;
-    
-}
-
-.grid-item22{
-    padding:  10px ;
-    overflow: hidden;
-    
-}
-
-.grid-item22 img{
- float: left;
- padding-right: 10px;
- 
-}
-
-.grid-item22 a{
-text-decoration: none;
-color: black;
-}
-.grid-item23{
-    border: 1px solid orange;
-    margin-top: 30px;
-    
-}
-.grid-item24{
-    border: 1px solid orange;
-    margin-top: 30px;
-    border-radius: 30px 30px 0px 0px;
-}
-
-.grid-item25{
-    background-color: #25ac45;
-    text-align: center;
-    font-size: 14px;
-    padding: 10px;
-    color: white;
-    border-radius: 20px;
-}
-.grid-item26{
-    padding: 20px;
-
-}
-.grid-item26 a{
-    text-decoration: none;
-    color: black;
-    padding-bottom: 10px;
-}
-.grid-item26 li{
-    list-style-type: initial;
-   
-    padding-bottom: 10px;
-}
-.grid-item3{
-    width: 406px;
-    height: 522px;
-    border: 1px solid orange;
-    border-radius: 30px 30px 0px 0px;
+  
+  /* Grid items */
+  .grid-item1, 
+  .grid-item2, 
+  .grid-item3, 
+  .grid-item4, 
+  .grid-item5, 
+  .grid-item6, 
+  .grid-item7, 
+  .grid-item8 {
+    border: 1px solid var(--border-color);
+    border-radius: var(--radius-large) var(--radius-large) 0 0;
     margin-top: 20px;
-    
-}
-.grid-item31{
-    
-    background-color: #25ac45;
+  }
+  
+  .grid-item1 {
+    grid-column: span 2;
+  }
+  
+  .grid-item2 {
+    grid-column: 3;
+    grid-row: span 5;
+  }
+  
+  .grid-item1 img, 
+  .grid-item3 img {
+    width: 100%;
+  }

```

Comment:
Consistent image padding might be necessary across '.grid-item1 img' and '.grid-item3 img' for uniform appearance.

```diff
  .grid-item1 img, 
  .grid-item3 img {
    width: 100%;
+   padding-left: 27px;
  }
```